<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PacketContentView</class>
 <widget class="QDialog" name="PacketContentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Packet Content</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Header Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="3">
       <widget class="QLabel" name="mib">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="mib_label">
        <property name="text">
         <string>MIB Version:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="apid_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="APID_label">
        <property name="text">
         <string>APID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="SequenceCount_label">
        <property name="text">
         <string>SequenceCount:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="DFH_label">
        <property name="text">
         <string>DFH present:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="ssc_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="type_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="PacketType_label">
        <property name="text">
         <string>PacketType:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="PI1_label">
        <property name="text">
         <string>PI1 Value:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="dfh_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Version_label">
        <property name="text">
         <string>Version:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="version_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="Sequence_label">
        <property name="text">
         <string>Sequence:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="sequence_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="PI1_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="PI2_label">
        <property name="text">
         <string>PI2 Value:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="PI2_value">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="SPID_label">
        <property name="text">
         <string>SPID:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="SPID">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="OBCCoreId_label">
        <property name="text">
         <string>OBC Core ID:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="core">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="OBSWbootCnt_label">
        <property name="text">
         <string>OBSW Boot Cnt:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="bootcnt">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Packet Data Field (hex)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bytes_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="data_line_edit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>data_line_edit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
